src/design-system/components/Prompt/Prompt.tsx

```diff
@@ -98,9 +102,10 @@ export const Prompt = ({
                   exit={{ opacity: 0 }}
                   transition={deceleratedShort}
                   background={background ?? 'surfaceMenu'}
-                  borderRadius="16px"
+                  borderRadius={borderRadius}
                   borderColor="separatorTertiary"
                   borderWidth="1px"
+                  style={{ overflow: 'hidden' }}
```

Comment:
This doesn't affect any other places where we use `<Prompt>` right ?

Reply:
no sirr